src/upscaler.py

```diff
@@ -703,7 +706,16 @@ def run(self):
 
                 # downscale frames with Lanczos
                 Avalon.info(_('Lanczos downscaling frames'))
-                shutil.rmtree(self.extracted_frames)
+                try:
+                    shutil.rmtree(self.extracted_frames)
+                except Exception:
+                    if platform.system() == 'Windows':
+                        for file in self.extracted_frames.iterdir():
+                            Avalon.info(_("Remove file:[%s] readOnly attribute." % file))
```

Comment:
Formatting should happen outside of the gettext function. Also, `%` operator is an outdated method. Use `.format`.
Also, please be consistent with your use of quotation marks. This project uses single quotes.
e.g., `Avalon.info(_('Removing read-only attribute from file {}').format(file))`

Reply:
Also are you sure that you'll want to print this out?
Even if you do, it shouldn't be `Avalon.info` but `Avalon.debug_info`.
This shouldn't have a vibrant color and get outputted to stdout.